MetalKitEssentials/MetalKitEssentials.sln

```diff
@@ -9,27 +9,17 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MetalKitEssentials.Mac", "M
 EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
-		Debug|iPhoneSimulator = Debug|iPhoneSimulator
 		Release|iPhone = Release|iPhone
-		Release|iPhoneSimulator = Release|iPhoneSimulator
 		Debug|iPhone = Debug|iPhone
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{3348FFBF-1401-4E10-AA27-718440E25D3F}.Debug|iPhone.ActiveCfg = Debug|Any CPU
 		{3348FFBF-1401-4E10-AA27-718440E25D3F}.Debug|iPhone.Build.0 = Debug|Any CPU

```

Comment:
MetalKit Essentials- 
1. mac sample is working as expected. 
2. ios-sample selects, Debug device and Release device. Do we need to change Debug -> Debug|iPhone, Release -> Release|iPhone. Otherwise build configuration for this project is correct.

https://screencast.com/t/mHMOHAQM.